modules/ossm-about-concepts-resources.adoc

```diff
@@ -0,0 +1,113 @@
+====
+
+You can access all `Istio` custom resource defintion (CRD) options through `spec.values` fields:
+
+.Example `Istio` resource CRD
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.22.3
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+----
+
+//Helm charts can only be used with istio-csr and Helm charts are a temporary work around for istio-csr. This might get confusing for users, so a NOTE was added.
+
+== `ConfigMap` and `MeshConfig`
+
+Istio uses a `ConfigMap` for its global configuration, called the `MeshConfig`. All of its settings are available through `spec.meshConfig`.
+
+To support canary updates of the control plane, {SMProduct} includes support for multiple Istio versions. You can select a version by setting  `spec.version` to the version you would like to install, prefixed with a v. You can update to a new version just by changing this field.
+
+{SMProduct} supports two different update strategies for your control planes:
+
+`InPlace`:: {SMProduct} Operator immediately replaces your existing control plane resources with the ones for the new version.
+
+`RevisionBased`:: Uses Istio's canary update mechanism by creating a second control plane to which you can migrate your workloads to complete the update.
+
+After creating an Istio resource, {SMProduct} generates a revision name for the resource based on the `updateStrategy`, and creates a corresponding `IstioRevision`.
+
+== `IstioRevision` resource
+
+The `IstioRevision` is a cluster-wide resource and the lowest-level API {SMProduct} provides. It is usually not created by the user, but by the operator itself. Its schema closely resembles that of the `Istio` resource - but instead of representing the state of a control plane you want to be present in your cluster, it represents a revision of that control plane.
+
+A revision of the control plane you want to be present in your cluster is an instance of Istio with a specific version and revision name, and its revision name can be used to add workloads or entire namespaces to the mesh. For example: by using the `istio.io/rev=<REVISION_NAME>` label.
+
+You can think of the relationship between the `Istio` and `IstioRevision` resources as similar to the relationship between Kubernetes' `ReplicaSet` and Pod: a `ReplicaSet` can be created by users and results in the automatic creation of Pods, which will trigger the instantiation of your containers.
+
+Similarly, users create an `Istio` resource which instructs the {SMProduct} Operator to create a matching `IstioRevision`, which then in turn triggers the creation of the Istio control plane. To do that, the {SMProduct} Operator will copy all of your relevant configuration from the `Istio` resource to the `IstioRevision` resource.
```

Comment:
This section does a good job of explaining how the two resources relate to each other!